lib/plug/basic_auth.ex

```diff
@@ -14,7 +14,7 @@ defmodule Plug.BasicAuth do
       import Plug.BasicAuth
       plug :basic_auth, username: "hello", password: "secret"
 
-  Or if you would rather put those in an environment variable:
+  Or if you would rather put those in a config file:
```

Comment:
I was on a fence here, now this section seems redundant to the section above, but if we're gonna use env variables we should use a different approach as they should be read at runtime.

Reply:
I meant to say app env. Good catch.